docs/application/native/guides/device/battery-monitor.md

```diff
@@ -0,0 +1,230 @@
+
+- Getting battery usage information of a particular application.
+
+  You can [retrieve information of an application](#appusage_get) by specifying its application ID, [resource](#resource_key) IDs, and the time [duration](#duration_key).
+
+- Getting battery usage information of a particular resource.
+
+  You can [retrieve information of a single or muiltiple resources](#resourceusage_get) by specifying its [resource](#resource_key) IDs, and the time [duration](#duration_key).
+
+## Prerequisites
+
+To enable your application to use the Battery Monitor functionality:
+
+1. To use the functions and data types of the Battery Monitor API (in [wearable](../../api/wearable/latest/group__CAPI__SYSTEM__BATTERY__MONITOR__MODULE.html) applications), include the `<battery_monitor.h>` header file in your application:
+
+   ```
+   #include <battery_monitor.h>
+   ```
+
+2. The application has to request permission by adding the following privileges to the `tizen-manifest.xml` file:
+
+   ```xml
+   <privileges>
+      <privilege>http://tizen.org/privilege/systemmonitor</privilege>
+   </privileges>
+   ```
+
+<a name="appusage_get"></a>
+## Getting Battery Usage Information of Particular Application
+
+The battery usage information contains a combination of application ID, resource ID, and time duration.
+
+1. Fetching information of an application for a particular resource over a certain duration of time.
+   The `battery_monitor_resource_id_e`, `battery_monitor_duration_type_e` enums can be used to provide information related to specific resource and particular duration type respectively, as shown in the following code:
+
+   ```
+   #include <battery_monitor.h>
+
+   void
+   func(void)
+   {
+	char* app_id = "org.tizen.samsung";
+
+	int error_code = BATTERY_MONITOR_ERROR_NONE;
+	int battery_usage = -1;
+	battery_monitor_resource_id_e resource_id = BATTERY_MONITOR_RESOURCE_ID_DISPLAY;
+	battery_monitor_duration_type_e duration_val = BATTERY_MONITOR_DURATION_TYPE_1DAY;
+
+	error_code = battery_monitor_get_usage_by_app_id_for_resource_id(app_id, resource_id, duration_val, &battery_usage);
+
+	if (error_code == BATTERY_MONITOR_ERROR_NONE)
+		printf("The Battery Usage for appid [%s], for resource display is [%d]", app_id, battery_usage);
+	else
+		printf("Error Occurred [%d]", error_code);
+
+	resource_id = BATTERY_MONITOR_RESOURCE_ID_WIFI;
+	duration_val = BATTERY_MONITOR_DURATION_TYPE_1WEEK;
+
+	error_code = battery_monitor_get_usage_by_app_id_for_resource_id(app_id, resource_id, duration_val, &battery_usage);
+
+	if (error_code == BATTERY_MONITOR_ERROR_NONE)
+		printf("The Battery Usage for appid [%s], for resource wifi is [%d]", app_id, battery_usage);
+
+	else
+		printf("Error Occurred [%d]", error_code);
+
+	free(app_id);
+
+	return;
+   }
+   ```
+
+2. Fetching total battery usage information of an application ID by combining all the resources over certain a certain duration of time.
```

Comment:
@mijin85cho Request your opinion on this: 

You can get information on total battery usage of an application ID by combining all the resources over a specific duration of time.
To  get a particular duration type, use the `battery_monitor_duration_type_e` enum.

Just to maintain consistency with other pages. (https://docs.tizen.org/application/native/guides/device/runtime#getting-application-runtime-information)

Reply:
@mijin85cho 
Get battery usage information of an application ID for a particular resource over a certain duration of time. To get a particular duration type, use the battery_monitor_duration_type_e enum.
@nawab-samsung 
You can get information on total battery usage of an application ID by combining all the resources over a specific duration of time.
To get a particular duration type, use the battery_monitor_duration_type_e enum.

Both the sentences convey my thoughts. I can use any version, which ever is more suitable according to Tizen guidelines.

Reply:
@nawab-samsung
I think it would have been better if you reference https://docs.tizen.org/application/native/guides/device/runtime#monitoring-runtime-information-changes, especially the sentences in the list.
("**_Use_** the runtime_info_set_changed_cb() function ~ " and "When no longer needed, **_deregister_** the callback with ~")

That "You can~" you referenced is the starting sentence, describing a section. 
And here, the sentences we are checking is under a list. They are not starting sentences. 


Also, I changed sentence a little bit because to me it was a little bit difficult to get what is the difference between the 2nd and the 3rd sections.

Suggested by you :
 - 2nd : You can get information on total battery usage of an application ID by combining all the resources over a specific duration of time.
 - 3rd : You can get the battery usage values for all the resources used by an application ID for a specific duration of time.

(It took some time to me to get what is the difference.)
I think to make it clear what is the main point of each section is required here.

Reply:
@mijin85cho 
Kindly suggest the best way and your opinion and we will take it as you say.